app/controllers/admin/reports/out_of_date_firms_controller.rb

```diff
@@ -1,24 +1,32 @@
-module Admin
-  module Reports
-    class OutOfDateFirmsController < Admin::ApplicationController
-      def index
-        @firms = Firm.joins('JOIN lookup_firms ON lookup_firms.fca_number = firms.fca_number')
-                 .where(parent: nil)
-                 .where('firms.registered_name != lookup_firms.registered_name')
-                 .order('firms.fca_number ASC')
-                 .pluck('firms.fca_number', 'lookup_firms.registered_name', 'firms.registered_name', 'firms.id')
-      end
+class Admin::Reports::OutOfDateFirmsController < Admin::ApplicationController
```

Comment:
What's the rationale for namespacing the class declaration as opposed to the nested moduling previously?

Reply:
Probably to keep the line length down. There are a few lines that are just under 80 chars, and if the modules were nested like before, the extra indenting would make them go over.